src/Aspire.Dashboard/Components/Controls/AspireMenuButton.razor.cs

```diff
@@ -43,6 +43,9 @@ public partial class AspireMenuButton : FluentComponentBase
     [Parameter]
     public string MenuButtonId { get; set; } = Identifier.NewId();
 
+    [Parameter]
+    public bool DisableIcon { get; set; }
```

Comment:
Is `HideIcon` a better name? Disable usually means something is still visible but it's in a disabled state.